Index: configure
===================================================================
--- configure
+++ configure	2009-11-09 23:40:42.000000000 -0300
@@ -741,11 +741,12 @@
 		} else {
 			proextra =
 			"CONFIG += qt crypto\n"
+			"INCLUDEPATH += /usr/include/QtCrypto\n"
 			"QT -= gui\n";
 		}
 
 		QString str =
-		"#include <QtCrypto>\n"
+		"#include <QtCrypto/QtCrypto>\n"
 		"\n"
 		"int main()\n"
 		"{\n"
@@ -2634,9 +2635,10 @@
 		if(!conf->INCLUDEPATH.isEmpty())
 			str += "INCLUDEPATH += " + conf->INCLUDEPATH + '\n';
 		if(!conf->LIBS.isEmpty())
-			str += "LIBS += " + conf->LIBS + '\n';
+			str += "LIBS += " + conf->LIBS + " -lqca" + '\n';
 		if(!conf->extra.isEmpty())
 			str += conf->extra;
+		str += "INCLUDEPATH += " + conf->INCLUDEPATH + "/usr/include/QtCrypto" + '\n';
 		str += '\n';
 
 		QByteArray cs = str.toLatin1();
